packages/pluggableWidgets/timeline-web/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "timeline-web",
   "widgetName": "Timeline",
-  "version": "1.0.1",
+  "version": "1.1.1",
```

Comment:
You need to change the version in the xml as well 😉

Reply:
Done. Thanks.